python/lsst/obs/fiberspectrograph/translator.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import os
+
+import astropy.units as u
+from astropy.time import Time
+
+from astro_metadata_translator import cache_translation
+from lsst.obs.lsst.translators.lsst import SIMONYI_TELESCOPE, LsstBaseTranslator
+
+from lsst.utils import getPackageDir
+
+__all__ = ["FiberSpectrographTranslator", ]
+
+log = logging.getLogger(__name__)
+
+
+class FiberSpectrographTranslator(LsstBaseTranslator):
+    """Metadata translator for Rubin calibration fibre spectrographs headers"""
+
+    name = "FiberSpectrograph"
+    """Name of this translation class"""
+
+    supported_instrument = "FiberSpec"
+    """Supports the Rubin calibration fibre spectrographs."""
+
+    default_search_path = os.path.join(getPackageDir("obs_fiberspectrograph"), "corrections")
+    """Default search path to use to locate header correction files."""
+
+    default_resource_root = os.path.join(getPackageDir("obs_fiberspectrograph"), "corrections")
+    """Default resource path root to use to locate header correction files."""
+
+    DETECTOR_MAX = 1
+
+    _const_map = {
+        "detector_num": 0,
+        "detector_name": "0",
+        "exposure_group": None,
```

Comment:
This will need to be filled in by something. It can't be None. I'm pretty sure the base translator will set that for you.

Reply:
How do you recommend "inheriting" the base `_const_map`?

Reply:
exposure_group is not const. It's defined in the parent class to just use the exposure_id if the GROUPID header is not present.